bin/configs/java-retrofit2-jackson.yaml

```diff
@@ -0,0 +1,8 @@
+generatorName: java
+outputDir: samples/client/petstore/java/retrofit2-jackson
+library: retrofit2-jackson
```

Comment:
Instead of creating another lib, what about using an option instead?

Ref: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java